src/list.js

```diff
@@ -407,8 +407,12 @@ Sk.builtin.list.prototype.list_sort_ = function (self, cmp, key, reverse) {
     var timsort;
     var has_key = key !== undefined && key !== null;
     var has_cmp = cmp !== undefined && cmp !== null;
-    if (reverse === undefined) {
-        reverse = false;
+    var rev;
+
+    if (reverse === undefined || reverse === Sk.builtin.none.$none) {
```

Comment:
Yeah I've found out some other things while I looked into this :smile: fix coming soon.